routers/usersRouter.js

```diff
@@ -0,0 +1,95 @@
+const usersDb = require("../data/helpers/usersDb");
+const bcrypt = require("bcryptjs");
+const customMw = require("../customMiddleware");
+
+const express = require("express");
+const router = express.Router();
+const protect = customMw.protect;
+const auth = customMw.auth;
+
```

Comment:
You could grab these from you `customMw` with destructuring 
```javascript 
const { protect, auth} = customMw
```
You're still passing in the `protect` over `customMw.protect`to your endpoints so the above suggestion would be more useful as you have more middleware to deal with 👍